pandas/core/reshape/pivot.py

```diff
@@ -482,11 +482,20 @@ def pivot(
     if columns is None:
         raise TypeError("pivot() missing 1 required argument: 'columns'")
 
-    columns_listlike = com.convert_to_list_like(columns)
+    # error: Argument 1 to "convert_to_list_like" has incompatible type "Hashable";
```

Comment:
does this suggest that the 'columns' annotation may be too loose?

Reply:
`convert_to_list_like` seems to be too strict: it literally accepts anything.